api/src/auth.js

```diff
@@ -11,7 +11,7 @@ const get = (path, headers) => {
     url: url.toString(),
     headers: headers,
     json: true
-  });
+  }).auth(environment.username, environment.password, false);
```

Comment:
The whole point of this request is to sent then authentication headers that the original request carries, not the ones for the admin user stored in env.

Reply:
If this fails, it means that your request headers are not valid user credentials. This is the desired behavior here.

Reply:
Couch is rejected anything I try on `_session`. I've tried posting from the haproxy server with a known valid auth token/cookie and it's rejecting.

Reply:
```

Reply:
I suspect this is an issue with your setup, and not a bug that affects CHT.

Reply:
Yeah, maybe. I've followed your settings, though. So, to be clear, you expect to be able to hit http://couchdb.org:5984/_session with nothing more than an auth token?

Reply:
yes, correct. 

```
curl 'http://127.0.0.1:6500/_session' -H 'Accept: application/json' --cookie 'AuthSession=YWRtaW46NjREQ0M2RDI6cOfK78w9qFb-XJyV1UWYtRPLV5o' 
{"ok":true,"userCtx":{"name":"admin","roles":["_admin","admin"]},"info":{"authentication_db":"_users","authentication_handlers":["cookie","default"],"authenticated":"cookie"}}
```

Reply:
Including with `require_valid_user = true` set?

I was half hoping it was a bug. This is driving me mad.

Reply:
yes, require_valid_user = true is essential for security... otherwise anyone can read heath data